### PR TITLE
Add english/defeat/

### PR DESCRIPTION
**Pull request by tomsmoker**

### CODE REVIEW

**Comment by tomsmoker** [comment]
@casey-collab, please merge this

File: english/defeat/defeat.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Just kidding.  Keep trucking on, solider.
```
Comment (casey-collab) [inline english/defeat/defeat.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by tomsmoker** [comment]
@casey-collab, please merge this

File: english/defeat/defeat.md (hunk 1 above)
Comment (casey-collab) [inline english/defeat/defeat.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**